.codecov.yml

```diff
@@ -0,0 +1 @@
+comment: false
```

Comment:
I think this will add a the basic config for status checks.

https://docs.codecov.com/docs/commit-status#basic-configuration

```suggestion
comment: false
coverage:
  status:
    project: true
```

Reply:
sounds good